src/components/Actions/actions.js

```diff
@@ -0,0 +1,69 @@
+import { CALL_API } from 'redux-api-middleware';
+import * as actionTypes from './actionTypes';
+
+export const fetchCar = () => ({
+  [CALL_API]: {
+    types: [
+      actionTypes.FETCH_CAR,
+      actionTypes.FETCH_CAR_SUCCESS,
+      actionTypes.FETCH_CAR_FAILURE,
+    ],
+    endpoint: 'http://apis.is/car?number=fma08',
+    method: 'GET',
+  },
+});
+
+export const fetchAuthenticationData = () => {
+  console.log('fetch auth data');
```

Comment:
remove unused console log, good example of an [eslint rule](https://eslint.org/docs/rules/no-console) you could add.

This rule will let you know if you have any eslint errors/warning, If you explicitly want to allow a specific console log you would have to do the following
```javascript
// eslint-disable-next-line
console.error('This is a useful error message that should print in the console if something major happens');
```